src/main/java/use_cases/display_availability_timeblocks/DisplayAvailabilityTimeBlockInteractor.java

```diff
@@ -0,0 +1,32 @@
+package use_cases.display_availability_timeblocks;
+import entity_layer.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class DisplayAvailabilityTimeBlockInteractor implements DisplayAvailabilityTimeBlockInputBoundary {
+    private DisplayAvailabilityTimeBlockOutputBoundary presenter;
+
+    public DisplayAvailabilityTimeBlockInteractor(DisplayAvailabilityTimeBlockOutputBoundary presenter) {
+        this.presenter = presenter;
+    }
+
+
+
+    @Override
+    public void displayAvailabilityTimeBlock() {
+        Schedule schedule = InMemoryUser.getActiveUser().getSchedule();
+        schedule.getAvailability();
+
+
+
+
+
+//        Use In Memory user class to get active user's curriculum.
```

Comment:
Solid progress here; once I'm done implementing the entities, you should be able to create and run this code! And good work on using inheritance through your interface to adhere to C.A.